torch/csrc/api/include/torch/nn/functional/distance.h

```diff
@@ -31,6 +31,14 @@ inline Tensor pairwise_distance(
       options.keepdim());
 }
 
+// ============================================================================
+
+/// Computes the p-norm distance between every pair of row vectors in the input.
+/// This function will be faster if the rows are contiguous.
+inline Tensor pdist(const Tensor& input, int64_t p = 2) {
```

Comment:
@jon-tow My sincere apologies - I made a mistake and just realized that `p` should be `double` type (https://en.wikipedia.org/wiki/Norm_(mathematics)#p-norm). Sorry for the inconvenience :(

Reply:
@yf225 Ah wow, good catch. I was also certain that `p` was integer rather than real.